internal/cmd/agent/deployer/summary/summary_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package summary_test
```

Comment:
Just curious about the name of this file: why not `suite_test.go` as done elsewhere?

Reply:
I'm fine with `suite_test.go` as well. The name `summary_suite_test.go` came directly from `ginkgo bootstrap`.